### ISSUE TITLE
Finalize requirements for the new community site

### ISSUE THREAD

**Issue by samccann** [ansible-community/community-team#163]

**Comment by Andersson007** [comment]
Created https://hackmd.io/PAiBqUSnSFaAt9w2UUPngg and reflected in it stuff from the initial google doc

**Comment by Andersson007** [comment]
Should it now be assigned to a person who announce it? (i'll be on PTO 13-17 March)

**Comment by Andersson007** [comment]
As @samccann merged both the hackmds to https://hackmd.io/PAiBqUSnSFaAt9w2UUPngg, I think we can move forward.
@samccann @cybette ideas?

**Comment by cybette** [comment]
Yes, thank you both for the efforts on this! I've created a friendly url for it and will share it :)

https://hackmd.io/@ansible-community/new-website-concept
